src/components/PageTVShows/PageTVShows.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const wrapperMain = {
+  padding: "120px 50px 50px 50px",
+};
```

Comment:
this code was duplicated several times. Please create Container component in the shared folder and use it everywhere

Reply:
Did this so that they stay together in the folder.

Reply:
Done

Reply:
it should be HOC component, not an object